composer.json

```diff
@@ -4,6 +4,11 @@
     "patches": {
       "drupal/search_api": {
         "Provide drush support to index a specific entity - https://www.drupal.org/project/search_api/issues/3016809": "https://www.drupal.org/files/issues/2020-02-08/search_api-index-specific-items-3016809-09.patch"
+      },
+      "drupal/key": {
+        "KeyProviderInterface::getKeyValue() doesn't always return a string - https://www.drupal.org/project/key/issues/3356052#comment-15030602": "https://git.drupalcode.org/project/key/-/merge_requests/11/diffs.patch",
+        "Deprecated Function: strlen(): Passing null to parameter #1 ($string) - https://www.drupal.org/project/key/issues/3323238": "https://www.drupal.org/files/issues/2023-02-01/3323238-6.patch",
+        "Error: Call to a member function getKeyValue on null - https://www.drupal.org/project/key/issues/3385523": "https://www.drupal.org/files/issues/2023-09-06/3385523-3.patch"
```

Comment:
Created this patch new. Please review.

Reply:
Added one comment on the patch over there, nothing major just being picky :P

Reply:
Yup see the point, thanks fro that. I have Updated the patch :)